app/viewmodels/journeys/taskList/TradeJourneyStatusesViewModel.scala

```diff
@@ -48,10 +48,14 @@ object TradeJourneyStatusesViewModel {
     val isAbroadAnswered = tradesJourneyStatuses.getStatusOrNotStarted(Abroad).isCompleted
     val incomeRow        = buildRow(Income, dependentJourneyIsFinishedForClickableLink = isAbroadAnswered)
 
+    val isCarAndAssetsBasedAllowanceAnswered = tradesJourneyStatuses.getStatusOrNotStarted(CapitalAllowancesTailoring).isCompleted
+    val adjustmentsRow = buildRow(Adjustments, dependentJourneyIsFinishedForClickableLink = isCarAndAssetsBasedAllowanceAnswered)
```

Comment:
dependent on expenses AND capital allowances

Reply:
Done :)